infra/terraform/test_fixtures/terraform-google-jenkins.tf

```diff
@@ -0,0 +1,95 @@
+locals {
+  jenkins_required_roles = [
+    "roles/compute.admin",
+    "roles/compute.networkAdmin",
+    "roles/iam.serviceAccountUser",
+  ]
+}
+
+resource "google_project" "jenkins" {
+  provider = "google.phoogle"
+
+  name            = "ci-jenkins"
+  project_id      = "ci-jenkins"
```

Comment:
This name isn't unique - choose something globally unique, like `phoogle-ci-jenkins`.

Reply:
Ok, thank you, I'll change it